example-workflows/gke-kustomize/README.md

```diff
@@ -0,0 +1,120 @@
+# Google Kubernetes Engine - GitHub Actions
+
+An example workflow that uses [GitHub Actions][actions] to deploy [a static
+website](site/) to an existing [Google Kubernetes Engine][gke] cluster.
+
+This code is intended to be an _example_. You will likely need to change or
+update values to match your setup.
+
+## Workflow description
+
+For pushes to the `main` branch, this workflow will:
+
+1.  Download and configure the Google [Cloud SDK][sdk] with the provided
+    credentials.
+
+1.  Build, tag, and push a container image to Google Artifact Registry.
+
+1.  Use a Kubernetes Deployment to push the image to the cluster.
+
+    - Note that a GKE deployment requires a unique Tag to update the pods. Using
+      a constant tag `latest` or a branch name `main` may result in successful
+      workflows that don't update the cluster.
+
+## Setup
+
+1.  Create a new Google Cloud Project (or select an existing project) and
+    [enable the Artifact Registry and Kubernetes Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=artifactregistry.googleapis.com,container.googleapis.com).
+
+1.  [Create a new GKE cluster][cluster] or select an existing GKE cluster.
+
+1.  Create or reuse a GitHub repository for the example workflow:
+
+    1.  [Create a repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository).
+
+    1.  Move into the repository directory:
+
+        ```
+        $ cd <repo>
+        ```
+
+    1.  Copy the example into the repository:
+
+        ```
+        $ cp -r <path_to>/github-actions/example-workflows/gke-kustomize/ .
```

Comment:
Nonblocking thought: In the future I wonder if there is an easier way to have this set up. Maybe we push to a non-main branch?

Reply:
Good point, will think about it